src/App.js

```diff
@@ -22,7 +22,11 @@ function App() {
 
     useEffect(() => {
         const init = async () => {
-            await initTranslation({lang: 'en-US'});
+            await initTranslation({
+                supportedLanguages: ['en-US'],
+                defaultLanguage: 'en-US',
+                currentLanguage: 'en-US'
```

Comment:
Keep in mind: This configuration object will in-fact change in a following PR as part of the second solution